backend/config/settings.py

```diff
@@ -121,8 +121,8 @@
 LOCALE_PATHS = ["./templates/locale"]
 
 TIME_ZONE = "Europe/Helsinki"
-USE_I18N = True
-USE_L10N = True
```

Comment:
Why disable these? 🤔

Reply:
I originally changed those so the returned error messages are not translated as stated in the commit message. But they are not needed anymore so I reverted the change.